token/cli/src/main.rs

```diff
@@ -4813,6 +5141,113 @@ fn app<'a, 'b>(
                 .arg(multisig_signer_arg())
                 .nonce_args(true)
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::DepositConfidentialTokens.into())
+                .about("Deposit amounts for confidential transfers")
+                .arg(
+                    Arg::with_name("token")
+                        .long("token")
+                        .validator(is_valid_pubkey)
+                        .value_name("TOKEN_MINT_ADDRESS")
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
```

Comment:
Does this work without the mint provided? Ie:
```suggestion
                        .required_unless("address")
```
Or is there an issue because of `index(2)` on the amount parameter?

Reply:
Hm yeah, I think this is a bit tricky unless we put `amount` before the mint. Not sure if it is worth breaking consistency with the rest of the commands.